src/Directory.Build.props

```diff
@@ -3,5 +3,6 @@
   <PropertyGroup>
     <AssemblyOriginatorKeyFile>..\Polly.snk</AssemblyOriginatorKeyFile>
     <SignAssembly>true</SignAssembly>
+    <LangVersion>9.0</LangVersion>
```

Comment:
Could just set to `latest`? Then we don't have to update it every release and it will just use whatever is stable in the SDK version in use.

Reply:
done